partitura/score.py

```diff
@@ -2826,6 +3071,26 @@ def validate(self):
             "down",
         ], f"Interval direction {self.direction} not found"
 
+    @property
+    def semitones(self):
+        return INTERVAL_TO_SEMITONES[self.quality + str(self.number)]
+
+    def change_quality(self, num):
```

Comment:
add documentation and comments to this function. It's not clear to me how it works